R/render.R

```diff
@@ -197,6 +197,33 @@ render_quarto_lang <- function(language_code, path, output_dir, type) {
   config_path <- file.path(temporary_directory, project_name, "_quarto.yml")
   config <- read_yaml(config_path)
 
+  freeze_folder_exists <- fs::dir_exists(
+    file.path(temporary_directory, project_name, "_freeze")
+  )
+
+  if (freeze_folder_exists) {
+    freeze_path <- fs::path(temporary_directory, project_name, "_freeze")
+    freeze_temp <- fs::path(
+      temporary_directory,
+      project_name, paste0("_freeze.", language_code)
+    )
+    freeze_ls <- fs::dir_ls(freeze_path, recurse = TRUE)
+
+    freeze_lang <- freeze_ls[grep(paste0("\\.", language_code, "$"), freeze_ls)]
```

Comment:
```suggestion
    freeze_lang <- purrr::keep(freeze_ls, \(x) grep(paste0("\\.", language_code, "$"), x))
```
maybe it needs language_code to be passed. This suggestion is just because it'd mean not repeating `freeze_ls` , and only a suggestion

Reply:
I'll take it. I'm usually so focused on using base R solutions that I don't look for ways to do things using the dependencies we have anyway. But I didn't know about `purrr::keep` until now, even though I'm a heavy user of {purrr}.